lib/enum/arrays.rb

```diff
@@ -1,24 +1,31 @@
 class Array
 
   def sum_of_numeric_elements
+    self.select { |element| element.is_a? Integer}.reduce(:+) + self.select { |element| element.is_a? Float}.reduce(:+)
```

Comment:
You can make this a lot shorter. First, you can combine the two selects with &&, like this: `select{ |element| element.is_a? Integer && element.is_a? Float?` Second, you can test both Integers and Floats by using Numeric instead, like this: `element.is_a? Numeric` Finally, you only need one reduce, at the very end. 

Putting it all together: `self.select{ |element| element.is_a? Numeric }.reduce(:+)`